backend/python/app/services/implementations/story_service.py

```diff
@@ -51,3 +54,33 @@ def create_story(self, story, content):
         db.session.refresh(new_story)
 
         return new_story
+
+    def create_translation(self, entity):
+        try:
```

Comment:
I'm on the fence about the nested try-excepts. What do you think about one try statement for the translation, and one try statement for the conents? if the contents throws, you can still delete the translation in the second except. 

Open to the nested try statements tho.